torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -630,6 +630,11 @@ def extra_repr(self):
         return f"inner_k_tiles={self.inner_k_tiles}"
 
 
+@dataclass(frozen=True)
+class Int4CPULayout(Layout):
+    def pre_process(self, input: torch.Tensor) -> torch.Tensor:
```

Comment:
you don't need to define this if it's the same as the default behavior?

you can just do `pass` here I think

Reply:
Yes, the default behavior is ok, use `pass` instead.